src/EditorFeatures/CSharpTest/Completion/CompletionProviders/XmlDocumentationCommentCompletionProviderTests.cs

```diff
@@ -760,6 +760,35 @@ static void Goo()
                 """, "cref", "langword", "href");
         }
 
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/72259")]
+        public async Task SeeAttributeNames2()
+        {
+            var text = """
+                class C
+                {
+                    /// <summary>
+                    /// <see l$$=""/>
+                    /// </summary>
+                    static void Goo()
+                    {
+                    }
+                }
+                """;
+            var expected = """
+                class C
+                {
+                    /// <summary>
+                    /// <see langword=""/>
```

Comment:
The test doesn't seem to show it, but I would expect the cursor to jump into quotes after completion in such scenario: `/// <see langword="$$"/>`. Please make sure the expected behavior here is fully tested

Reply:
The new test now tests for this and I actually had to implement this